### PR TITLE
Wdlupdate

### PR DESCRIPTION
**Pull request by bshifaw**
- Added more descriptive ReadMe
- Added optional default docker and gatk path
- Switched command to use picard tool in gatk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+
+#### Outputs:
+ - Set of .txt files containing the validation reports, one per input file
+
+### Cromwell version support 
+ - Successfully tested on v32
+ - Does not work on versions < v23 due to output syntax
+
+ Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+ For program versions, see docker containers.
```
Comment (katevoss) [inline README.md]: Is "docker containers" supposed to be a link? If not, I would add clarification of where someone can find the docker containers.

File: validate-bam.wdl [hunk 2]
```diff
@@ -54,21 +60,24 @@ workflow ValidateBamsWf {
...
+      --INPUT ${bam_file} \
+      --OUTPUT ${output_name} \
+      --MODE ${validation_mode}
   }
   runtime {
-    docker: "broadinstitute/genomes-in-the-cloud:2.2.3-1469027018"
+    docker: docker
     memory: mem_size
     cpu: "1"
     disks: "local-disk " + disk_size + " HDD"
```
Comment (katevoss) [inline validate-bam.wdl:83]: Note in order to get this WDL to run, I had to [increase the bootDiskSizeGb](https://gatkforums.broadinstitute.org/gatk/discussion/comment/49453#Comment_49453) as the GATK image is currently very large.

**Review by katevoss — APPROVED** [review]
This looks good! Just two small comments, but I was able to run the WDL successfully.

File: validate-bam.wdl (hunk 2 above)
Comment (bshifaw) [reply validate-bam.wdl:83]: The latest gatk release (4.0.6.0) has reduced its image size, this should longer be a problem.

**Pull request merged**